gui-wm/wayfire/wayfire-0.4.0.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="compiz like 3D wayland compositor"
+HOMEPAGE="https://github.com/WayfireWM/wayfire"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/WayfireWM/${PN}.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/WayfireWM/${PN}/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm64 ~x86"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="+gles2 +wf-config +wlroots +elogind systemd debug"
+
+DEPEND="
+	dev-libs/libevdev
+	dev-libs/libinput
+	gui-libs/gtk-layer-shell
+	media-libs/glm
+	media-libs/mesa:=[gles2,wayland,X]
+	media-libs/libjpeg-turbo
+	media-libs/libpng
+	media-libs/freetype:=[X]
+	x11-libs/libdrm
+	x11-libs/gtk+:3=[wayland,X]
+	x11-libs/cairo:=[X,svg]
+	x11-libs/libxkbcommon:=[X]
+	x11-libs/pixman
+	gles2? ( media-libs/libglvnd[X] )
+	wf-config? ( ~gui-libs/wf-config-${PV}[debug=] )
+	!wf-config? ( !gui-libs/wf-config )
+	wlroots? ( >=gui-libs/wlroots-0.10.0[elogind=,systemd=,X] )
+	!wlroots? ( !gui-libs/wlroots )
+"
+
+RDEPEND="
+		${DEPEND}
+		elogind? ( sys-auth/elogind )
```

Comment:
Based on the IUSE, is elogind required or just the default choice by yourself as maintainer? Would make it difficult for systemd users to have to negate elogind then add systemd.

Reply:
It was my choice, because it makes life simpler for me.

Now that I think about it, the gnome profile already adds elogind.

Good point, I will change this to nothing by default (same as sway) :smile_cat:

Reply:
fixed

Reply:
Sway has a REQUIRED_USE setup for elogind or systemd. I would recommend this as the best option. Then profiles or the user can set it accordingly.